docker/server/entrypoint.sh

```diff
@@ -65,7 +65,12 @@ do
     # check if variable not empty
     [ -z "$dir" ] && continue
     # ensure directories exist
-    if ! mkdir -p "$dir"; then
+    if [ "$DO_CHOWN" = "1" ]; then
+      make_dir="mkdir -p"
+    else
+      make_dir="$gosu mkdir -p"
+    fi
+    if ! $make_dir "$dir"; then
```

Comment:
I'd suggest using `$mkdir` here to make it searchable

```suggestion
    if [ "$DO_CHOWN" = "1" ]; then
      mkdir="mkdir -p"
    else
      mkdir="$gosu mkdir -p"
    fi
    if ! $mkdir "$dir"; then
```

Reply:
I've specifically made it not the same name to avoid confusion. But I'm ok with changing it.